include/libpmemobj++/experimental/radix_tree.hpp

```diff
@@ -629,6 +664,29 @@ radix_tree<Key, Value, BytesView>::~radix_tree()
 	}
 }
 
+/**
+ * Checks whether the container is empty.
+ *
+ * @return true if container is empty, false otherwise.
+ */
+template <typename Key, typename Value, typename BytesView>
+bool
+radix_tree<Key, Value, BytesView>::empty() const noexcept
+{
+	return size_ == 0;
+}
+
+/**
+ * @return maximum number of elements the container is able to hold due to PMDK
+ * limitations.
+ */
+template <typename Key, typename Value, typename BytesView>
+typename radix_tree<Key, Value, BytesView>::size_type
+radix_tree<Key, Value, BytesView>::max_size() const noexcept
+{
+	return PMEMOBJ_MAX_ALLOC_SIZE / sizeof(value_type);
```

Comment:
For radix tree it;s actually more than that. Elements are not holds in a contigious memory range, so you can just return std::numeric_limits<difference_type>::max();